src/gui/folderman.cpp

```diff
@@ -1206,21 +1206,14 @@ void FolderMan::removeFolder(Folder *f)
     f->removeFromSettings();
 
     unloadFolder(f);
-    if (currentlyRunning) {
```

Comment:
This code was probably meant to ensure we upload everything.  However it might come from the days where there was just one folder.

Reply:
Hmmm
https://github.com/owncloud/client/commit/c1045af500d18c52129d2a81f9ac51ec7ca92c8b

Reply:
Well I guess it really can be removed.